src/pages/add-lock-exception/AddLockExceptionForm/OrganisationUnitSelection/SelectByLevel.js

```diff
@@ -0,0 +1,139 @@
+import { useDataEngine, useAlert } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+import Controls from './Controls'
+import useOrgUnitCache from './use-org-unit-cache'
+
+const currentRootOrgUnitsQuery = {
+    orgUnits: {
+        resource: 'organisationUnits',
+        id: ({ currentRootId }) => currentRootId,
+        params: ({ level }) => ({
+            fields: 'id,path',
+            level,
+            paging: false,
+        }),
+    },
+}
+const orgUnitsQuery = {
+    orgUnits: {
+        resource: 'organisationUnits',
+        params: ({ level }) => ({
+            fields: 'id,path',
+            level,
+            paging: false,
+        }),
+    },
+}
+
+const SelectByLevel = ({ levels, currentRoot, onSelect, onDeselect }) => {
```

Comment:
This component is a rewrite of https://github.com/dhis2/d2-ui/blob/master/packages/org-unit-select/src/OrgUnitSelectByLevel.component.js